zng/type.go

```diff
@@ -62,6 +63,28 @@ var (
 	TypeEnum     = &TypeOfEnum{}
 )
 
+const (
+	IdBool     = 0
+	IdByte     = 1
+	IdInt16    = 2
+	IdUint16   = 3
+	IdInt32    = 4
+	IdUint32   = 5
+	IdInt64    = 6
+	IdUint64   = 7
+	IdFloat64  = 8
+	IdString   = 9
+	IdBytes    = 10
+	IdBstring  = 11
+	IdEnum     = 12
+	IdIP       = 13
+	IdPort     = 14
+	IdNet      = 15
+	IdTime     = 16
+	IdDuration = 17
+	IdAny      = 18
```

Comment:
Maybe sort these for readability.

Reply:
They are sorted! :-)

Reply:
We can bikeshed sort order of various constant names and align the spec and the code around the outcome... later PR

Reply:
👍